src/user_templates_api/templates/jupyter_lab/templates/squidpy/metadata.json

```diff
@@ -8,5 +8,13 @@
     ], 
     "is_multi_dataset_template": true,
     "template_format": "jinja",
+    "examples": [
+        {
+            "example_id": 0,
+            "description": "Look at the neturophil cluster of this slide-seq dataset and examine its Ripley's L-function",
+            "dataset_type": ["salmon_rnaseq_slideseq"],
```

Comment:
Where is this value coming from?

Reply:
For templates that only work on certain types, I have added a dataset_type property in one of the code cells, and then check that all the datasets are of any of the acceptable types. I also have this for 'required_filetypes', where a dataset needs to have all of the required filetypes. 

Perhaps it doesn't make sense to add these (as the selected datasets should be correct), but I think it's good to signal to the user what kind of datasets / files a template can be used with.

Reply:
@lchoy should we change to assay_display_name? Or should this still be called data_type? Let me know

Reply:
`assay_display_name` is more user friendly and matches the text used in search.